aims/aim3.md

```diff
@@ -0,0 +1,60 @@
+While image data have been extensively modeled, less is known about the learning capacity of such models applied to single cell gene expression data.
+The _objective of this aim_ is to assess the extent to which single cell gene expression latent spaces can be manipulated and how such manifold preserving properties will enable biological discovery regarding cell type, cellular differentiation, and disease perturbation.
+The _working hypothesis of this aim_ is that the latent space will preserve arithmetic operations between state transitions of differentiating hematopoietic cells.
+
+## Preliminary Data:
+
+We have previously modeled bulk gene expression data with denoising autoencoders and non-linear activation functions [4–6].
+These models have engineered features that encode information about gene ontology terms and KEGG pathways.
+We have shown that these models capture more information than linear models like PCA, ICA, and NMF, and that the learned features are often overlapping and redundant [7].
+We have also recently trained a VAE on cancer gene expression datasets and observed that the learned latent features capture interesting biological distributions of features across different tumors [8].
+Therefore, we are confident that modeling gene expression with reconstruction cost captures meaningful biology.
+Furthermore, we trained VAEs and GANs on electronic health record (EHR) data including a model to generate privacy aware fake datasets to supplement learning or data sharing [9,10].
+We will develop similar approaches to extract knowledge from Human Cell Atlas (HCA) single cell data.
+
+## Enabling latent space arithmetic in single cell gene expression with variational autoencoders:
+
+Once a VAE is trained on HCA single cell gene expression data, we will obtain the latent space representation of a heterogeneous population of blood cells.
+We will use well characterized gene expression markers and unsupervised clustering algorithms to define core subsets of common blood cell types.
+We will define core subsets by positive silhouette widths of well characterized cell types representing a variety of the differentiation trajectories of hematopoiesis.
+We will test our hypothesis that the learned latent space vectors can be manipulated mathematically to obtain biological insights and useful properties.
+We will calculate the mean latent space vector representing erythroid cells and mean vector representing the population of stem cell progenitors.
+We will take the mean of these two vectors, which, if the hypothesis were true, will represent cells in intermediate trajectories.
+We will repeat this procedure with mature leukocyte populations.
+We will determine how similar the observed mean vectors for observed intermediate cell-types are to the predicted cell types.
```

Comment:
yes, this is similar to one of @jaclyn-taroni's concerns [here](https://github.com/greenelab/czi-rfa/pull/3#discussion_r133245275)

I think I have a clearer picture of what type of experiments to propose here and with what cell types (fibroblast --> cardiomyocyte lineage)